docs/sources/setup/install/helm/reference.md

```diff
@@ -5200,203 +5200,8 @@ null
-  "podSecurityContext": {
-    "fsGroup": 10001,
-    "runAsGroup": 10001,
-    "runAsNonRoot": true,
-    "runAsUser": 10001
-  },
-  "querier": {},
-  "query_range": {},
-  "query_scheduler": {},
-  "readinessProbe": {
-    "httpGet": {
-      "path": "/ready",
-      "port": "http-metrics"
-    },
-    "initialDelaySeconds": 30,
-    "timeoutSeconds": 1
-  },
-  "revisionHistoryLimit": 10,
-  "rulerConfig": {},
-  "runtimeConfig": {},
-  "schemaConfig": {},
-  "server": {
-    "grpc_listen_port": 9095,
-    "http_listen_port": 3100,
-    "http_server_read_timeout": "600s",
-    "http_server_write_timeout": "600s"
-  },
-  "serviceAnnotations": {},
-  "serviceLabels": {},
-  "storage": {
-    "azure": {
-      "accountKey": null,
-      "accountName": null,
-      "connectionString": null,
-      "endpointSuffix": null,
-      "requestTimeout": null,
-      "useFederatedToken": false,
-      "useManagedIdentity": false,
-      "userAssignedId": null
-    },
-    "filesystem": {
-      "chunks_directory": "/var/loki/chunks",
-      "rules_directory": "/var/loki/rules"
-    },
-    "gcs": {
-      "chunkBufferSize": 0,
-      "enableHttp2": true,
-      "requestTimeout": "0s"
-    },
-    "s3": {
-      "accessKeyId": null,
-      "backoff_config": {},
-      "endpoint": null,
-      "http_config": {},
-      "insecure": false,
-      "region": null,
-      "s3": null,
-      "s3ForcePathStyle": false,
-      "secretAccessKey": null,
-      "signatureVersion": null
-    },
-    "swift": {
-      "auth_url": null,
-      "auth_version": null,
-      "connect_timeout": null,
-      "container_name": null,
-      "domain_id": null,
-      "domain_name": null,
-      "internal": null,
-      "max_retries": null,
-      "password": null,
-      "project_domain_id": null,
-      "project_domain_name": null,
-      "project_id": null,
-      "project_name": null,
-      "region_name": null,
-      "request_timeout": null,
-      "user_domain_id": null,
-      "user_domain_name": null,
-      "user_id": null,
-      "username": null
-    },
-    "type": "s3"
-  },
-  "storage_config": {
-    "boltdb_shipper": {
-      "index_gateway_client": {
-        "server_address": "{{ include \"loki.indexGatewayAddress\" . }}"
-      }
-    },
-    "hedging": {
-      "at": "250ms",
-      "max_per_second": 20,
-      "up_to": 3
-    },
-    "tsdb_shipper": {
-      "index_gateway_client": {
-        "server_address": "{{ include \"loki.indexGatewayAddress\" . }}"
-      }
-    }
-  },
-  "structuredConfig": {},
-  "tenants": [],
-  "testSchemaConfig": {
-    "configs": [
-      {
-        "from": "2024-04-01",
-        "index": {
-          "period": "24h",
-          "prefix": "index_"
-        },
-        "object_store": "{{ include \"loki.testSchemaObjectStore\" . }}",
-        "schema": "v13",
-        "store": "tsdb"
-      }
-    ]
-  },
-  "tracing": {
-    "enabled": false
-  },
-  "useTestSchema": false
-}
+			<td><pre lang="">
+See values.yaml
```

Comment:
```suggestion
See [values.yaml](https://github.com/grafana/loki/blob/main/production/helm/loki/values.yaml)
```

Reply:
that `reference.md` file is always regenerated. We'd have to instead add the link it on `values.yaml` in such a way that the generated `reference.md` would generate the desired link.